cmd/finch/nerdctl_windows.go

```diff
@@ -107,21 +106,250 @@ func handleVolume(systemDeps NerdctlCommandSystemDeps, args []string, index int)
+		before, v, found = strings.Cut(prefix, "=")
+	} else {
+		if (index + 1) < len(args) {
+			v = args[index+1]
+		} else {
+			return fmt.Errorf("invalid positional parameter for %s", prefix)
+		}
+	}
+
+	entries := strings.Split(v, ",")
+	m := make(map[string]string)
+	for _, e := range entries {
+		parts := strings.Split(e, "=")
+		m[strings.TrimSpace(parts[0])] = strings.TrimSpace(parts[1])
+	}
+	sp, ok := m["src"]
+	if !ok {
+		return nil
+	}
+	wslPath, err := convertToWSLPath(systemDeps, sp)
+	if err != nil {
+		return err
+	}
+	m["src"] = wslPath
+
+	// Convert to string representation
+	s := mapToString(m)
+
+	if found {
+		args[index] = fmt.Sprintf("%s=%s", before, s)
+	} else {
+		args[index+1] = s
+	}
+
+	return nil
+}
+
+// cp command handler, takes command arguments and converts hostpath to wsl path in place. It ignores all other arguments
+func cpHandler(systemDeps NerdctlCommandSystemDeps, cmdArgs []string) error {
+	for i, arg := range cmdArgs {
+		// -L and --follow-symlink don't have to be processed
+		if arg == "-" || strings.HasPrefix(arg, "-") {
+			continue
+		} else {
+			// If argument contains container path, then continue
+			colon := strings.Index(arg, ":")
+
+			// this is a container path
+			if colon > 1 {
+				continue
+			}
+			wslPath, err := convertToWSLPath(systemDeps, arg)
+			if err != nil {
+				return err
+			}
+			cmdArgs[i] = wslPath
+		}
+	}
+	return nil
+}
+
+// this is the handler for image build command. It translates build context to wsl path.
+func imageBuildHandler(systemDeps NerdctlCommandSystemDeps, cmdArgs []string) error {
+	var err error
+	if cmdArgs[len(cmdArgs)-1] != "--debug" {
+		cmdArgs[len(cmdArgs)-1], err = convertToWSLPath(systemDeps, cmdArgs[len(cmdArgs)-1])
+		if err != nil {
+			return err
+		}
+	} else {
+		cmdArgs[len(cmdArgs)-2], err = convertToWSLPath(systemDeps, cmdArgs[len(cmdArgs)-2])
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
 var aliasMap = map[string]string{
 	"build": "image build",
+	"save":  "image save",
+	"load":  "image load",
 	"cp":    "container cp",
 	"run":   "container run",
 }
 
 var argHandlerMap = map[string]map[string]argHandler{
 	"image build": {
-		"-f":     handleFilePath,
-		"--file": handleFilePath,
+		"-f":        handleFilePath,
+		"--file":    handleFilePath,
+		"--iidfile": handleFilePath,
+		"-o":        handleOutputOption,
+		"--output":  handleOutputOption,
+		"--secret":  handleSecretOption,
+	},
+	"image save": {
+		"-o":       handleFilePath,
+		"--output": handleFilePath,
+	},
+	"image load": {
+		"-i":      handleFilePath,
+		"--input": handleFilePath,
 	},
 	"container run": {
 		"--label-file": handleFilePath,
 		"--cosign-key": handleFilePath,
+		"--cidfile":    handleFilePath,
 		"-v":           handleVolume,
 		"--volume":     handleVolume,
+		"--mount":      handleBindMounts,
 	},
+	"compose": {
+		"--file": handleFilePath,
+	},
+}
+
+var commandHanderMap = map[string]commandHandler{
```

Comment:
Are there any 3 word mappings?

Reply:
There are no commands which have 2 subcommands that require filepath handling. The only nerdctl command that has 2 subcommands is [nerdctl-ipfs-registry-serve](https://github.com/containerd/nerdctl/blob/main/docs/command-reference.md#nerd_face-nerdctl-ipfs-registry-serve) but no filepath handling in it. In the future if there were 3 subcommands that required handling, we'll have to add them as commands to https://github.com/runfinch/finch/blob/0e2dbf9d936795cc373802357200a52a001a82a4/cmd/finch/nerdctl.go#L374.